Setup.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
```

Comment:
We have changed `Setup.bat` file recently with some workflow improvements. In order keep the two scripts consistent, update this file to correspond to our changes. You can check them [here](https://github.com/spatialos/UnrealGDK/commits/master/Setup.bat) (changes from https://github.com/spatialos/UnrealGDK/pull/544 onwards).
Also, could you please add a comment:
```# EXPERIMENTAL!
# This file is experimental and is not maintained directly by Improbable. Please use at your own risk.
```